tools/codegen/codespec/api_to_provider_spec_mapper.go

```diff
@@ -82,11 +82,14 @@ func apiSpecResourceToCodeSpecModel(oasResource APISpecResource, resourceConfig
 	updateOp := oasResource.UpdateOp
 	readOp := oasResource.ReadOp
 
-	createPathParams := pathParamsToAttributes(createOp)
-	createRequestAttributes := opRequestToAttributes(createOp)
-	updateRequestAttributes := opRequestToAttributes(updateOp)
-	createResponseAttributes := opResponseToAttributes(createOp)
-	readResponseAttributes := opResponseToAttributes(readOp)
+	// TODO remove useCustomNestedTypes usage once typing is supported for all nested attributes: object, list, set & map - CLOUDP-349095
```

Comment:
As part of this clean up task can we make sure to also remove redundant conversion logic? Maybe a separate CLOUDP capturing these cleanup aspects can be good.